extensions/simplelogin/src/list_alias.tsx

```diff
@@ -1,13 +1,14 @@
 import { loadAllAliases, updateAliasPinnedStatus, deleteAlias, toggleAliasState } from "./api/simplelogin_api";
 import { useEffect, useMemo, useState } from "react";
 import { Action, ActionPanel, Icon, List, confirmAlert, showToast, Toast, Alert } from "@raycast/api";
+import { useCachedState } from "@raycast/utils";
 import { AliasResponse } from "./models/alias";
 import moment from "moment";
 
-type Filter = "all" | "" | "pinned" | "others";
+type Filter = "all" | "" | "pinned" | "others" | "with-description" | "without-description";
 
 export default function Command() {
-  const [aliases, setAliases] = useState<AliasResponse[]>([]);
+  const [aliases, setAliases] = useCachedState<AliasResponse[]>("aliases", []);
```

Comment:
This is the caching mechanism I introduced here; after the initial fetch, aliases are cached between command restarts.

Reply:
Makes sense 🙂